<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Utahime Timer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63, 63, 63);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_timer_v3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>V3 timer</string>
    </property>
    <widget class="QLabel" name="label_timer_v3_countup">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>80</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>55</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_timer_v3_countdown">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>55</pointsize>
      </font>
     </property>
     <property name="text">
      <string>30.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_v3_current_indicator">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_v3_current_indicator</zorder>
    <zorder>label_timer_v3_countdown</zorder>
    <zorder>label_timer_v3_countup</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton_timer_v3_start">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>250</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: rgb(255, 255, 255);
  background-color: rgba(255, 255, 255, 0);
  border-radius:6px;
  border-color: rgb(255, 255, 255);
  border-style:solid;
  border-width:2px;
 
}

 
QPushButton:hover:pressed {
  background-color: rgb(210, 210, 210);
}
 

QPushButton:hover:!pressed {
  background-color: rgba(223, 223, 223, 120);
}</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>周速</string>
    </property>
    <widget class="QLabel" name="label_timer_lap_countup">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>35</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_laptime_log1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_laptime_log2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_laptime_log5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_laptime_average">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_laptime_log3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_laptime_log4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>15:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5周ログ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5周平均</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>時速</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_speed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>メイリオ</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>960</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>万/h</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_timer_lap">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: rgb(255, 255, 255);
  background-color: rgba(255, 255, 255, 0);
  border-radius:6px;
  border-color: rgb(255, 255, 255);
  border-style:solid;
  border-width:2px;
 
}

 
QPushButton:hover:pressed {
  background-color: rgb(210, 210, 210);
}
 

QPushButton:hover:!pressed {
  background-color: rgba(223, 223, 223, 120);
}</string>
    </property>
    <property name="text">
     <string>Lap</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_timer_v3_shortcut_key">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>310</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-bottom: 1px solid rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_timer_lap_shortcut_key">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-bottom: 1px solid rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Ctrl+L</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ショートカット</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ショートカット</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>241</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>設定</string>
    </property>
    <widget class="QRadioButton" name="radioButton_30s_sound_enable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>30 s 時音声再生ON</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_30s_sound_file_path">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
border-bottom: 1px solid rgb(255,255,255);</string>
     </property>
     <property name="text">
      <string>sound.mp3</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_30s_sound_file_dialog">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  color: rgb(255, 255, 255);
  background-color: rgba(255, 255, 255, 0);
  border-radius:6px;
  border-color: rgb(255, 255, 255);
  border-style:solid;
  border-width:2px;
 
}

 
QPushButton:hover:pressed {
  background-color: rgb(210, 210, 210);
}
 

QPushButton:hover:!pressed {
  background-color: rgba(223, 223, 223, 120);
}</string>
     </property>
     <property name="text">
      <string>・・・</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>v3タイマーstartショートカットキー</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QKeySequenceEdit" name="keySequenceEdit_timer_v3_shortcut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_timer_v3_shortcut_fix">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  color: rgb(255, 255, 255);
  background-color: rgba(255, 255, 255, 0);
  border-radius:6px;
  border-color: rgb(255, 255, 255);
  border-style:solid;
  border-width:2px;
 
}

 
QPushButton:hover:pressed {
  background-color: rgb(210, 210, 210);
}
 

QPushButton:hover:!pressed {
  background-color: rgba(223, 223, 223, 120);
}</string>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>周速ショートカットキー</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_timer_lap_shortcut_fix">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>270</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  color: rgb(255, 255, 255);
  background-color: rgba(255, 255, 255, 0);
  border-radius:6px;
  border-color: rgb(255, 255, 255);
  border-style:solid;
  border-width:2px;
 
}

 
QPushButton:hover:pressed {
  background-color: rgb(210, 210, 210);
}
 

QPushButton:hover:!pressed {
  background-color: rgba(223, 223, 223, 120);
}</string>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QKeySequenceEdit" name="keySequenceEdit_timer_lap_shortcut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QKeySequenceEdit" name="keySequenceEdit_timer_v3_reverse_shortcut">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>v3裏打ちショートカットキー</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_timer_v3_reverse_shortcut_fix">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  color: rgb(255, 255, 255);
  background-color: rgba(255, 255, 255, 0);
  border-radius:6px;
  border-color: rgb(255, 255, 255);
  border-style:solid;
  border-width:2px;
 
}

 
QPushButton:hover:pressed {
  background-color: rgb(210, 210, 210);
}
 

QPushButton:hover:!pressed {
  background-color: rgba(223, 223, 223, 120);
}</string>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>20</width>
      <height>321</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_timer_v3_reverse">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>250</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  color: rgb(255, 255, 255);
  background-color: rgba(255, 255, 255, 0);
  border-radius:6px;
  border-color: rgb(255, 255, 255);
  border-style:solid;
  border-width:2px;
 
}

 
QPushButton:hover:pressed {
  background-color: rgb(210, 210, 210);
}
 

QPushButton:hover:!pressed {
  background-color: rgba(223, 223, 223, 120);
}</string>
    </property>
    <property name="text">
     <string>裏打ち</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ショートカット</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_timer_v3_reverse_shortcut_key">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-bottom: 1px solid rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
